aptos-move/framework/aptos-framework/doc/staking_config.md

```diff
@@ -786,4 +1347,21 @@ The maximum_stake must be greater than maximum_stake in the range of Specified s
 </code></pre>
 
 
+
+<a name="@Specification_1_validate_rewards_config"></a>
+
+### Function `validate_rewards_config`
+
+
+<pre><code><b>fun</b> <a href="staking_config.md#0x1_staking_config_validate_rewards_config">validate_rewards_config</a>(rewards_rate: u64, min_rewards_rate: u64, rewards_rate_denominator: u64, rewards_rate_period_in_micros: u64, rewards_rate_decrease_rate_bps: u64)
+</code></pre>
+
+
+Abort at any condition in StakingRewardsConfigValidationAborts.
+
+
+<pre><code><b>include</b> <a href="staking_config.md#0x1_staking_config_StakingRewardsConfigValidationAbortsIf">StakingRewardsConfigValidationAbortsIf</a>;
+</code></pre>
+
+
 [move-book]: https://move-language.github.io/move/introduction.html
```

Comment:
Same comment as above: Can we update this in the source file to instead go to the new location?
https://aptos.dev/guides/move-guides/book/SUMMARY

Reply:
Sure, created a separated PR to address this issue: https://github.com/aptos-labs/aptos-core/pull/7425

Reply:
Thank you! LGTM. :-D